README.md

```diff
@@ -23,16 +23,34 @@ are running the simulation from.
 
 ## Running
 
+### Run with python virtual environment
+
 With the simulation package installed, you should have a new command-line
-program called `simulation ` available.  Try running `simulation --help` to get
+program called `nlisim ` available.  Try running `nlisim --help` to get
 more information.  To run a simulation, you will need configure a configuration.
 There is an example configuration in the repository to get you started.
 
-Now run simulation up to 50 seconds using the first example config.
+Now run simulation up to 50 timesteps using the first example config.
 ```bash
     nlisim --config config.ini.example run 50
 ```
 
+### Run with docker container
+
+Build the docker image.
```

Comment:
Building locally defeats the benefits of having a pre-built image published on Docker Hub. I'll update the example to implicitly pull the published version (which `docker run` will do by default).